src/App/pages/ReposSearchPage/ReposSearchPage.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+
+import Avatar from "@components/Avatar";
+import Button from "@components/Button";
+import Input from "@components/Input";
+import ReposBranchesDrawer from "@components/ReposBranchesDrawer";
+import RepoTile from "@components/RepoTile";
+import SearchIcon from "@components/SearchIcon";
+import "./ReposSearchPage.css";
+import { getOrganizationReposListFetch } from "@root/root";
+import { RepoItem } from "src/store/GitHubStore/types";
+
+const ReposSearchPage = () => {
+  const [value, setValue] = useState("");
+  const [isLoading, setIsLoading] = useState(false);
+  const [repos, setRepos] = useState<RepoItem[]>([]);
+  const [selectedRepo, setSelectedRepo] = useState<RepoItem>();
+  const [visible, setVisible] = useState(false);
+
+  const handlerInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setValue(event.target.value);
+  };
+
+  const handlerButton = (event: React.MouseEvent) => {
+    event.preventDefault();
+    setIsLoading(true);
+    getOrganizationReposListFetch(value).then((value) => {
+      if (value) {
+        setRepos(value);
+      } else {
+        alert("Что-то пошло не так!");
+      }
+    });
+    setValue("");
+    setIsLoading(false);
+  };
+
+  const handlerRepo = (event: React.MouseEvent, item: RepoItem) => {
+    setSelectedRepo(item);
+    setVisible(true)
+  };
+
+  const handlerDrawer = () => {
+      setVisible(false)
+  };
+
+  return (
+    <div>
+      <form className="search-line">
+        <Input value={value} onChange={handlerInput} />
+        <Button disabled={isLoading} onClick={handlerButton}>
+          <SearchIcon />
+        </Button>
+      </form>
+      {repos.map((el) => {
+        return (
```

Comment:
```suggestion
       {repos.map((el) => (
             <div/>
       )}
```

сделай стрелочную  = читаемость